usr/share/rear/prep/default/320_include_uefi_env.sh

```diff
@@ -63,15 +63,20 @@ local esp_proc_mounts_line=()
 #   /dev/sda1 /boot/efi vfat rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=ascii,shortname=mixed,utf8,errors=remount-ro 0 0
 # cf. https://github.com/rear/rear/issues/2095#issuecomment-475684942
 # and https://github.com/rear/rear/issues/2095#issuecomment-481739166
-# The ESP could be mounted on /boot/efi or on /boot.
+# The ESP could be mounted on /boot/efi or on /boot or even /efi.
+# cf. https://wiki.archlinux.org/title/EFI_system_partition#Typical_mount_points
 # First try /boot/efi:
 esp_proc_mounts_line=( $( grep ' /boot/efi ' /proc/mounts || echo false ) )
 if is_false $esp_proc_mounts_line ; then
-    # If nothing is mounted on /boot/efi try /boot:
-    esp_proc_mounts_line=( $( grep ' /boot ' /proc/mounts || echo false ) )
+    # If nothing is mounted on /boot/efi try /efi:
+    esp_proc_mounts_line=( $( grep ' /efi ' /proc/mounts || echo false ) )
```

Comment:
I guess this can be simplified via checking for a list of "known" EFI mount points.